style.css

```diff
@@ -1,9 +1,29 @@
+label {
```

Comment:
Make labels invisible for their purpose is to make the website more accessible for people using screen readers